staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher.go

```diff
@@ -822,7 +822,7 @@ func setCachingObjects(event *watchCacheEvent, versioner storage.Versioner) {
 			// event.PrevObject is used to deliver DELETE watch events and
 			// for them, we set resourceVersion to <current> instead of
 			// the resourceVersion of the last modification of the object.
-			updateResourceVersionIfNeeded(object.object, versioner, event.ResourceVersion)
+			updateResourceVersion(object, versioner, event.ResourceVersion)
```

Comment:
just noticed we're now passing in `object` instead of `object.object`... does `Update resource version of the underlying object` need to be reworded?

Reply:
changed